api/pet/petRouter.js

```diff
@@ -0,0 +1,396 @@
+const express = require('express');
+const router = express.Router({ mergeParams: true });
+
+// Models
+const petsDb = require('../pet/petModel');
+
+// Utilities
+const { errDetail } = require('../utils/utils');
+
+// Auth middleware
+const authRequired = require('../middleware/authRequired');
+
+/**
+ * @swagger
```

Comment:
Props for writing documentation for each route.

I've never used swagger before and I'm curious to see what it looks like when compiled.